render/wgpu/src/context3d/mod.rs

```diff
@@ -286,6 +295,29 @@ impl WgpuContext3D {
                 .as_ref()
                 .expect("Missing compiled pipeline"),
         );
+        if let Some(rect) = &self.scissor_rectangle {
+            let current_size = self.current_texture_size.unwrap();
+            if rect.x_min.to_pixels() as u32 > current_size.width
+                || rect.x_max.to_pixels() as u32 > current_size.width
+                || rect.y_min.to_pixels() as u32 > current_size.height
+                || rect.y_max.to_pixels() as u32 > current_size.height
```

Comment:
Should this check also happen for negative x/y?

Reply:
I've adjusted the check